TestKit/Sources/TestKit/XCTestCase+Wait.swift

```diff
@@ -72,7 +72,7 @@ extension XCTestCase {
     public func waitFor<ValueType>(file: StaticString = #file,
                                    line: UInt = #line,
                                    timeout: TimeInterval = 5.0,
-                                   await: @escaping (_ promise: (@escaping (ValueType) -> Void)) throws -> Void) rethrows -> ValueType {
+                                   awaiting: @escaping (_ promise: (@escaping (ValueType) -> Void)) throws -> Void) rethrows -> ValueType {
```

Comment:
The compiler recommends replacing the `await` word, to avoid confusion with the new `await` keyword in Swift 5.5's async/await. The other option would have been to escape it with backticks, but I think replacing makes more sense here.